packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -69,6 +74,9 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
       _positionController.reverse();
   }
 
+  /// The color that should be used in the active state.
```

Comment:
maybe `active state (i.e. when [value] is true)` ? (and similarly for inactiveColor)